docs/network-flow-visibility.md

```diff
@@ -819,11 +826,120 @@ storage size, please modify the `sizeLimit` in the following section.
+```
+
+If you prefer not to clone the repository and create a customized manifest manually,
+please follow the steps below to deploy the ClickHouse with Persistent Volumes:
+
+1. Provision the `PersistentVolume`. K8s supports a great number of
+[`PersistentVolume` types](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes).
+You can provision your own `PersistentVolume` per your requirements. Here are
+two simple examples for your reference.
+
+    - Local PV allows you to store the ClickHouse data at a pre-defined path on
+    a specific node. Refer to [createLocalPv.yml][local_pv_yaml] to create the
+    PV. Please replace `LOCAL_PATH` with the path to store the ClickHouse data
+    and label the node used to store the ClickHouse data with
+    `clickhouse/instance=data`.
+
+    - NFS PV allows you to store the ClickHouse data on an existing NFS server.
+    Refer to [createNfsPv.yml][nfs_pv_yaml] to create the PV. Please replace
+    `NFS_SERVER_ADDRESS` with the host name of the NFS server and `NFS_SERVER_PATH`
+    with the exported path on the NFS server.
+
+    In both examples, you can set `.spec.capacity.storage` in `PersistentVolume`
+    to your storage size. This value is for informative purpose as K8s does not
+    enforce the capacity of PVs. If you want to limit the storage usage, you need
+    to ask for your storage system to enforce that. For example, you can create
+    a Local PV on a partition with the limited size. We recommend using a dedicate
+    saving space for the ClickHouse if you are going to run the Flow Collector in
+    production.
+
+    As these examples do not use any dynamical provisioner, the reclaim policy
+    for the PVs are `Retain` by default. After stopping the Grafana Flow Collector,
+    if you no long need the data for future use, you may need to manually clean
+    up the data on the local disk or NFS server.
+
+1. Request the PV for the ClickHouse. Please add a `volumeClaimTemplate` section
+under `.spec.templates` to the resource `ClickHouseInstallation` in
+`flow-visibility.yml` as the sample below. `storageClassName` should be filled
+as your own `StorageClass` name, and `.resources.requests.storage` should be set
+to your storage size.
+
+    ```yaml
+    volumeClaimTemplates:
+    - name: clickhouse-storage-template
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 8Gi
+        storageClassName: clickhouse-storage
+    ```
+
+    Then add this template as `dataVolumeClaimTemplate` to the section below.
+
+    ```yaml
+    defaults:
+      templates:
+        dataVolumeClaimTemplate: clickhouse-storage-template
+        podTemplate: pod-template
+        serviceTemplate: service-template
+    ```
+
+1. Remove the in-memory deployement related contents, which is defined as
```

Comment:
rephrase this sentence: Remove the in-memory related deployment options, by removing the appropriate volume and volumeMount for the `ClickHouseInstallation` resource in `flow-visibility.yml`.